build-system/tasks/karma.conf.js

```diff
@@ -208,6 +208,7 @@ module.exports = {
     failOnConsoleError: !process.env.TRAVIS && !process.env.LOCAL_PR_CHECK,
     // TODO(rsimha, #14432): Set to false after all tests are fixed.
     captureConsole: true,
+    debugConsole: false,
```

Comment:
Let's call this `verboseLogging`, since it's triggered by the `--verbose` flag.

Reply:
sure. renamed